docs/assistant/cloud/libraries/using-library-assets/entire-libraries.md

```diff
@@ -0,0 +1,13 @@
+---
+id: entire-libraries
+title: Adding entire libraries to a site
+sidebar_label: Entire Libraries
+---
+
+In addition to adding single items to Assistant Pro libraries, you can also import entire libraries.  To import an entire library:
```

Comment:
In addition to adding single items to Assistant Pro libraries, you can also import entire libraries.  ->  You can import an entire Assistant Pro library.